scripts/docs/features-html-output/build-features-html-output.js

```diff
@@ -0,0 +1,571 @@
+	const numberOfPackages = parseMetadataFiles();
+
+	console.log( `✨ ${ chalk.green( `The features HTML output from ${ numberOfPackages } packages has been generated successfully.` ) }` );
+
+	const shouldCommitChanges = process.argv.includes( '--commit' );
+
+	if ( shouldCommitChanges ) {
+		commitChanges();
+	}
+} catch ( error ) {
+	console.log( `❌ ${ chalk.red( 'An error occurred during parsing a package metadata file.' ) }` );
+	console.log( error );
+}
+
+/**
+ * Main parser function. Its purpose is to:
+ * - read all package metadata files,
+ * - parse and prepare the data for generating the features HTML output overview,
+ * - use the parsed data to create tables for each package, that contains all plugins and their possible HTML output.
+ *
+ * Returns total number of parsed files.
+ *
+ * Each generated table contains 2 columns: "Plugin" and "HTML output". Each table cell in the "Plugin" column has a human-readable name of
+ * the plugin (which is a link to the feature documentation) and the name of the class used to create the plugin (which is a link to the API
+ * documentation). For each row in the "Plugin" column there is at least one row in the "HTML output" column. If given plugin does not
+ * generate any output, the one and only row in the "HTML output" column contains the word "None". Each item from the `htmlOutput` property
+ * from the package metadata file corresponds to a separate row in the "HTML output" column. It contains one or more preformatted paragraphs
+ * describing the possible HTML output: HTML elements, their CSS classes, inline styles, other attributes and comments.
+ *
+ * ┏━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+ * ┃    Plugin    ┃           HTML output          ┃
+ * ┣━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┫
+ * ┃ first plugin │ output #1 for the first plugin ┃
+ * ┃              ├────────────────────────────────┨
+ * ┃              ┄                                ┄
+ * ┃              ├────────────────────────────────┨
+ * ┃              │ output #N for the first plugin ┃
+ * ┃──────────────┼────────────────────────────────┨
+ * ┄              ┄                                ┄
+ * ┃──────────────┼────────────────────────────────┨
+ * ┃ last plugin  │ output #1 for the last plugin  ┃
+ * ┃              ├────────────────────────────────┨
+ * ┃              ┄                                ┄
+ * ┃              ├────────────────────────────────┨
+ * ┃              │ output #N for the last plugin  ┃
+ * ┗━━━━━━━━━━━━━━┷━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
+ *
+ * Generated table is preceded by the package name as a heading and the link to a source package metadata file on GitHub.
+ *
+ * @returns {Number}
+ */
+function parseMetadataFiles() {
+	const parsedFiles = parseFiles()
+		.map( packageMetadata => {
+			const outputRows = packageMetadata.plugins
+				.map( plugin => {
+					const numberOfRowsPerPlugin = plugin.htmlOutput.length;
+
+					const pluginNameRowspan = numberOfRowsPerPlugin > 1 ?
```

Comment:
`cellRowspan`? I think that this variable is not connected with the plugin name at all.